pyproject.toml

```diff
@@ -266,6 +264,9 @@ ignorelist = [
     "locals",
 ]
 
+[tool.ruff.lint.flake8-errmsg]
+max-string-length = 0
```

Comment:
fwiw, this is already the default value: https://docs.astral.sh/ruff/settings/#lint_flake8-errmsg_max-string-length